### PR TITLE
Add .test suffix to unit test files

### PR DESCRIPTION
**Pull request by zguesmi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## vNEXT
 - Migrate unit test files to Typescript & Hardhat:
-    - ENSIntegration (#105)
+    - ENSIntegration (#105, #106)
```
Comment (gfournieriExec) [inline CHANGELOG.md:5]: Why put this in the line of ENS integration ?

**Review by gfournieriExec — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zguesmi) [reply CHANGELOG.md:5]: Just because it's in the scope of the migration. I'm open for suggestions, where else do you suggest?

Comment (gfournieriExec) [reply CHANGELOG.md:5]: Could just add on top of ENS integration "reanme testing files" (?)

Comment (zguesmi) [reply CHANGELOG.md]: Yes [Update changelog](https://github.com/iExecBlockchainComputing/PoCo/pull/106/commits/6d09be4fca7c52f0c925c43c26ce822b9abfee96)

**Pull request merged**